### PR TITLE
Update in Changing the PIN

### PR DESCRIPTION
**Pull request by joinimran**
Made some update in Changing the PIN.
Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9475

### CODE REVIEW

File: windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-use-virtual-smart-cards.md [hunk 1]
```diff
@@ -80,8 +80,12 @@ A TPM-based virtual smart card is labeled **Security Device** in the user interf
 
 ## Changing the PIN
 
-The PIN for virtual smart card can be changed by pressing Ctrl+Alt+Del, and then selecting the TPM virtual smart card under **Sign in options**.
-
+The PIN for virtual smart card can be changed by following steps:
+- Log on with the old pin or password.
```
Comment (illfated) [inline windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-use-virtual-smart-cards.md:84]: PIN = personal identification number (not the noun "pin")
```suggestion
- Log on with the old PIN or password.
```

File: windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-use-virtual-smart-cards.md [hunk 2]
```diff
@@ -80,8 +80,12 @@ A TPM-based virtual smart card is labeled **Security Device** in the user interf
...
+- Press Ctrl+Alt+Del and choose **Change a password**.
+- Click ""Sign-in Options**.
+- Click the **Virtual smart card icon**.
+- Change the pin.
```
Comment (illfated) [inline windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-use-virtual-smart-cards.md:88]: (as above)
```suggestion
- Change the PIN.
```

File: windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-use-virtual-smart-cards.md [hunk 3]
```diff
@@ -80,8 +80,12 @@ A TPM-based virtual smart card is labeled **Security Device** in the user interf
 
 ## Changing the PIN
 
-The PIN for virtual smart card can be changed by pressing Ctrl+Alt+Del, and then selecting the TPM virtual smart card under **Sign in options**.
-
+The PIN for virtual smart card can be changed by following steps:
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-use-virtual-smart-cards.md:83]:
```suggestion
The PIN for a virtual smart card can be changed by following these steps:
```

File: windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-use-virtual-smart-cards.md [hunk 4]
```diff
@@ -80,8 +80,12 @@ A TPM-based virtual smart card is labeled **Security Device** in the user interf
...
+- Log on with the old pin or password.
+- Press Ctrl+Alt+Del and choose **Change a password**.
+- Click ""Sign-in Options**.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-use-virtual-smart-cards.md:86]:
```suggestion
- Select **Sign-in Options**.
```

File: windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-use-virtual-smart-cards.md [hunk 5]
```diff
@@ -80,8 +80,12 @@ A TPM-based virtual smart card is labeled **Security Device** in the user interf
...
+- Click the **Virtual smart card icon**.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-use-virtual-smart-cards.md:87]:
```suggestion
- Select the virtual smart card icon.
```

File: windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-use-virtual-smart-cards.md (hunk 1 above)
Comment (JohanFreelancer9) [inline windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-use-virtual-smart-cards.md:84]:
```suggestion
- Sign in with the old PIN or password.
```

File: windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-use-virtual-smart-cards.md (hunk 2 above)
Comment (JohanFreelancer9) [inline windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-use-virtual-smart-cards.md:88]:
```suggestion
- Enter and confirm the new PIN.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Pull request merged**